client/my-sites/media-library/style.scss

```diff
@@ -142,45 +142,14 @@
 		margin-top: 2px;
 	}
 
-	.gridicons-chevron-down {
-		top: -3px;
-	}
-
 	.gridicon {
 		top: 4px;
 	}
 
-	&:after {
-		@include noticon( '\f431', 20px );
-		line-height: 16px;
-		color: rgba( $gray, 0.5 );
-		padding-top: 6px;
-	}
-
-	@include breakpoint( "<480px" ) {
-		.gridicons-chevron-down {
-			width: 18px;
-			height: 18px;
-			color: rgba( $gray, 0.5 );
-		}
-
-		.gridicon {
-			top: 2px;
-		}
-
-		&:after {
-			padding-top: 3px;
-		}
-	}
-}
-
-.media-library__source-button.is-open.is-borderless {
-	&:after {
-		@include noticon( '\f432', 20px );
-	}
-
-	.gridicon {
-		color: $gray-dark;
+	.gridicons-chevron-down {
+		width: 18px;
+		height: 18px;
```

Comment:
Why do we need to specify the height/width here?

Reply:
If it's not there, then this bit of css overrides the 18px size. I think it's in the button component css:

![screen shot 2018-03-06 at 4 24 24 pm](https://user-images.githubusercontent.com/618551/37062072-062c4d6a-215b-11e8-89ea-e84778da8aa6.png)

Reply:
Oh yea, I've run in to this before.